examples/01_basic/bolt_pretension.py

```diff
@@ -22,516 +22,871 @@
+    return contact_region
 
-SURFACE4 = GEOM.Children[3].Children[0]
-SURFACE4.Material = "Steel"
 
-SURFACE5 = GEOM.Children[4].Children[0]
-SURFACE5.Material = "Steel"
+# %%
+# Set the friction coefficient, small sliding, and update stiffness settings for the contact region
+def advanced_contact_settings(
+    contact_region: Ansys.ACT.Automation.Mechanical.Connections.ContactRegion,
+    friction_coefficient: int,
+    small_sliding: ContactSmallSlidingType,
+    update_stiffness: UpdateContactStiffness,
+) -> None:
+    """Set the friction coefficient, small sliding, and update stiffness settings for the
+    contact region.
+
+    Parameters
+    ----------
+    contact_region : Ansys.ACT.Automation.Mechanical.Connections.ContactRegion
+        The contact region to set the settings for.
+    friction_coefficient : int
+        The friction coefficient for the contact region.
+    small_sliding : ContactSmallSlidingType
+        The small sliding setting for the contact region.
+    update_stiffness : UpdateContactStiffness
+        The update stiffness setting for the contact region.
+    """
+    contact_region.FrictionCoefficient = friction_coefficient
+    contact_region.SmallSliding = small_sliding
+    contact_region.UpdateStiffness = update_stiffness
 
-SURFACE6 = GEOM.Children[5].Children[0]
-SURFACE6.Material = "Steel"
 
 # %%
-# Define coordinate system
-# ~~~~~~~~~~~~~~~~~~~~~~~~~
+# Add a command snippet to the contact region with the specified Archard Wear Model
+def add_command_snippet(
+    contact_region: Ansys.ACT.Automation.Mechanical.Connections.ContactRegion,
+    archard_wear_model: str,
+) -> None:
+    """Add a command snippet to the contact region with the specified Archard Wear Model.
+
+    Parameters
+    ----------
+    contact_region : Ansys.ACT.Automation.Mechanical.Connections.ContactRegion
+        The contact region to add the command snippet to.
+    archard_wear_model : str
+        The Archard Wear Model command snippet to add to the contact region.
+    """
+    contact_region_cmd = contact_region.AddCommandSnippet()
+    contact_region_cmd.AppendText(archard_wear_model)
 
-coordinate_system = CS_GRP.AddCoordinateSystem()
-coordinate_system.OriginDefineBy = CoordinateSystemAlignmentType.Fixed
-coordinate_system.OriginX = Quantity(-195, "mm")
-coordinate_system.OriginY = Quantity(100, "mm")
-coordinate_system.OriginZ = Quantity(50, "mm")
-coordinate_system.PrimaryAxis = CoordinateSystemAxisType.PositiveZAxis
 
 # %%
-# Define Contacts
-# ~~~~~~~~~~~~~~~
-# Change contact settings
+# Add and define contact regions
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 # %%
-# Delete existing contacts
-
-for connection in CONN_GRP.Children:
+# Set up the model connections and delete the existing connections for ConnectionGroups
+connections = model.Connections
+for connection in connections.Children:
     if connection.DataModelObjectCategory == DataModelObjectCategory.ConnectionGroup:
         connection.Delete()
```

Comment:
obj.Delete() is obsolete. We should use DataModel.Remove(obj) or DataModel.Remove([obj,... ])